packages/executor/src/services/EntryPointService/versions/0.0.7.ts

```diff
@@ -70,11 +78,22 @@ export class EntryPointV7Service implements IEntryPointService {
           this.networkConfig.estimationGasLimit
         )
       : undefined;
+    
+    const estimateCallGasArgs: CallGasEstimationProxy.EstimateCallGasArgsStruct = {
+      userOp: packUserOp(userOp),
+      isContinuation: true,
+      maxGas: "100000",
```

Comment:
maxGas should not be hardcoded to `100k`. Simplest nft mint will consume more, we don't want those operations to revert